tests/test_virtualscreening.py

```diff
@@ -175,19 +178,19 @@ def test_vs_similarity():
 
     vs = virtualscreening(n_cpu=1)
     vs.load_ligands('sdf', xiap_actives_docked)
-    vs.similarity('ifp', cutoff=0.8, query=ref_mol, protein=receptor)
+    vs.similarity('ifp', cutoff=0.9, query=ref_mol, protein=receptor)
```

Comment:
Same here - probably rebasing artifact, but if it passes I'm fine with it.

Reply:
no, actually this one was on purpose because with 0.8 we don't find anything with the current version

Reply:
hm, now using 0.8 actually works. I don't know what was the reason it didn't, but I'll get back to 0.8 anyway